test-suite/overnightindexedswap.cpp

```diff
@@ -202,6 +202,7 @@ void testBootstrap(bool telescopicValueDates,
     Natural paymentLag = 2;
 
     std::vector<ext::shared_ptr<RateHelper> > estrHelpers;
+    Handle<Quote> spread(ext::shared_ptr<Quote>(new SimpleQuote(0.00)));
```

Comment:
You can use `auto spread = makeQuoteHandle(0.0)` instead.  Or you can use `make_shared` instead of `new`.  People might be looking at the tests as examples, so we're trying to show good manners 😄